logging/logging.yml

```diff
@@ -0,0 +1,51 @@
+version: "3"
+
+networks:
+
+  elk:
+    external: false
+
+  proxy:
```

Comment:
I'd make two files. One should be logging stack (ELK + LogSpout) without the proxy. Not everyone is using DFP (even though they should). The other can be logging-df-proxy.yml with the same content as this file.